### ISSUE TITLE
Telnet

### ISSUE THREAD

**Issue by sany2** [jeelabs/esp-link#572]
Hi. Is it possible to send data from the esp console to a PC with a telnet server installed?

**Comment by LionBS76** [comment]
> Hi. Is it possible to send data from the esp console to a PC with a telnet server installed?

Hello.
As i understand esp-link is a server TCP/UDP to Serial. Esp-link can't be a client for telnet server.

**Comment by uzi18** [comment]
Yes client mode is not implemented on webserver interface but ...
You can connect any Arduino compatible MCU and use it to open connection and send data
